thymleafdemo/src/main/java/com/luv2code/springboot/thymleafdemo/service/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.luv2code.springboot.thymleafdemo.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.luv2code.springboot.thymleafdemo.dao.EmployeeRepository;
+import com.luv2code.springboot.thymleafdemo.entity.Employee;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService {
+
+	private EmployeeRepository employeeRepository;
```

Comment:
@avantibonde can be immutable, use final. Not sure will work with DI or not
```suggestion
	private final EmployeeRepository employeeRepository;
```